### ISSUE TITLE
Extra indentation added on wrapped constructor arguments

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1260]
I'm seeing the formatter change this (wrapped lines indented 1 tab more than first):
```
                         devicePositions[device] = rect = new Rect(
                             Vector2.Lerp(rect.position, target.position, 0.1f),
                             Vector2.Lerp(rect.size, target.size, 0.1f));
```
to this (wrapped lines indented 3 tabs more than first):
```
                         devicePositions[device] = rect = new Rect(
                                     Vector2.Lerp(rect.position, target.position, 0.1f),
                                     Vector2.Lerp(rect.size, target.size, 0.1f));
```

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60012-UNI-12303.cs" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60012-UNI-12303.cs"
index 01be829..29bd878 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60012-UNI-12303.cs"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60012-UNI-12303.cs"
@@ -1,4 +1,4 @@
 //This gets a double indentation
 devicePositions[device] = rect = new Rect(
-    Vector2.Lerp(rect.position, target.position, 0.1f),
-    Vector2.Lerp(rect.size, target.size, 0.1f));
+            Vector2.Lerp(rect.position, target.position, 0.1f),
+            Vector2.Lerp(rect.size, target.size, 0.1f));
```

Debug support files:
---
[60012-UNI-12303.cs](https://gist.githubusercontent.com/mihaipopescu/54907d6308471f0a0d11ef86f5cab7fb/raw/01be82912166012377b82027c7f43c251f1b3aa3/60012-UNI-12303.cs)
[60012-UNI-12303.cs.log](https://gist.githubusercontent.com/mihaipopescu/54907d6308471f0a0d11ef86f5cab7fb/raw/e42e9a051854408dbaabb9ee1c061effd879068c/60012-UNI-12303.cs.log)
[60012-UNI-12303.cs.unc](https://gist.githubusercontent.com/mihaipopescu/54907d6308471f0a0d11ef86f5cab7fb/raw/431a4afc39b6132cb8c0f6d68642f782c41c71a0/60012-UNI-12303.cs.unc)
[Uncrustify.CSharp.cfg](https://gist.githubusercontent.com/mihaipopescu/54907d6308471f0a0d11ef86f5cab7fb/raw/1d064c1fabb76ce328dd015ed40023207de20ca1/Uncrustify.CSharp.cfg)
[UNI-12303.cs](https://gist.githubusercontent.com/mihaipopescu/54907d6308471f0a0d11ef86f5cab7fb/raw/01be82912166012377b82027c7f43c251f1b3aa3/UNI-12303.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
Caused by:
```sql
indent_align_assign             = false
```

**Comment by mihaipopescu** [comment]
I don't get where we have an extra indentation from. Is it the `new` or the `=` ?

I think it could be from `=` as we have another related issue (not reported yet):

```cpp
    m_Map[VeryLongEnumKey_1].ptr
        = m_Map[VeryLongEnumKey_2].ptr
                = m_Map[VeryLongEnumKey_3].ptr
                        = m_Map[VeryLongEnumKey_4].ptr
                                = m_Map[VeryLongEnumKey_5].ptr
                                        = m_Map[VeryLongEnumKey_6].ptr;
```

where all these assignments are indented when we want them without indent.

**Comment by gmaurel** [comment]
Fixed at db6adb6c69b238746c60233a423eac3df02c4073, Dec 20 20:25:54 2017
